src/Chapter14/Listing14.14.DeclaringAGenericDelegateType.cs

```diff
@@ -4,7 +4,10 @@
 
     public class Thermostat
     {
-        public delegate void EventHandler<TEventArgs>(object sender, TEventArgs e)
+        #region INCLUDE
+        public delegate void EventHandler<TEventArgs>(
+            object sender, TEventArgs e)
+            #endregion INCLUDE
             where TEventArgs : EventArgs;
```

Comment:
I think we probably want to move the `#endregion` after the semicolon, otherwise it looks funny and is not valid C#.
```suggestion
        public delegate void EventHandler<TEventArgs>(
            object sender, TEventArgs e)
            where TEventArgs : EventArgs;
        #endregion INCLUDE
```